css/bodystyle.css

```diff
@@ -0,0 +1,200 @@
+* {
+  padding: 0;
+  margin: 0;
+  text-decoration: none;
+  list-style: none;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: white;
+}
+
+nav {
+  background-color: white;
+  height: 60px;
+  width: 100%;
+}
+
+img.logo {
+  width: 20px;
+  height: auto;
+  margin: 10px;
+  margin-left: 40px;
+}
+
+nav ul {
+  float: right;
```

Comment:
Please don't use `float` rules in your CSS. Float is the old way of making layouts in CSS and it has now been replaced by Flexbox (which was introduced a few years ago). 
Set `display: flex` on the parent of the element instead.